### PR TITLE
DOC: added examples to mean and fixed quotes

### PR DESCRIPTION
**Pull request by rscheiwe**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Comment by rscheiwe** [comment]
Ok, fixed those!

File: pandas/core/generic.py [hunk 1]
```diff
@@ -67,7 +67,8 @@
     args_transpose='axes to permute (int or label for object)',
     optional_by="""
         by : str or list of str
-            Name or list of names to sort by""")
+            Name or list of names to sort by
```
Comment (jreback) [inline pandas/core/generic.py]: why are you changing all of this?

**Review by jreback — CHANGES_REQUESTED** [review]
cc @datapythonista

File: pandas/core/generic.py (hunk 1 above)
Comment (rscheiwe) [reply pandas/core/generic.py]: Apologies...That was in response to the validate. Let me push jus tthe mean additions from my local.

**Comment by mroeschke** [comment]
Why was `doc/test.parquet` committed? That should have been out of scope of this PR?

**Comment by rscheiwe** [comment]
I'll have to restore and restart. I'm not sure why that pushed.

**Pull request closed without merging**